sympy/functions/special/polynomials.py

```diff
@@ -1203,8 +1214,7 @@ class assoc_laguerre(OrthogonalPolynomial):
         Degree of Laguerre polynomial. Must be ``n >= 0``.
 
     alpha : Expr
-        Arbitrary expression. For ``alpha=0`` regular Laguerre
-        polynomials will be generated.
+        Arbitrary expression. For ``alpha=0`` regular Laguerre polynomials will be generated.
```

Comment:
This line crosses 80 characters. You don't need to change this. 
It was done so that the code follows PEP-8(https://www.python.org/dev/peps/pep-0008/)

Reply:
Okay sure